code/modules/mob/living/simple_mob/subtypes/vore/rabbit.dm

```diff
@@ -0,0 +1,150 @@
+/datum/category_item/catalogue/fauna/rabbit		//TODO: VIRGO_LORE_WRITING_WIP
+	name = "Wildlife - Rabbit"
+	desc = ""
+	value = CATALOGUER_REWARD_TRIVIAL
+
+/mob/living/simple_mob/vore/rabbit
+	name = "rabbit"
+	desc = "It's a fluffy, smol bunny! Adorable!"
+	tt_desc = "Vulpes zerda"
+
+	icon_state = "rabbit_brown"
+	icon_living = "rabbit_brown"
+	icon_dead = "rabbit_brown_dead"
+	icon_rest = "rabbit_brown_rest"
+	icon = 'icons/mob/vore.dmi'
+
+	faction = "rabbit"
+	maxHealth = 30
+	health = 30
+
+	response_help = "pats"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+
+	harm_intent_damage = 5
+	melee_damage_lower = 1
+	melee_damage_upper = 3
+	attacktext = list("nipped")
+	
+	movement_cooldown = 3
+
+	say_list_type = /datum/say_list/rabbit
+	ai_holder_type = /datum/ai_holder/simple_mob/passive
+	
+	// Vore vars
+	vore_active = 1
+	vore_bump_chance = 10
+	vore_bump_emote	= "playfully lunges at"
+	vore_pounce_chance = 40
+	vore_pounce_maxhealth = 100 // They won't pounce by default, as they're passive. This is just so the nom check succeeds. :u
+	vore_default_mode = DM_HOLD
+	vore_icons = SA_ICON_LIVING
+	
+	var/body_color 				//brown, black and white, leave blank for random
+	
+	var/grumpiness 				// This determines how grumpy we are. Pet us to increase it, leave us alone to decrease.
```

Comment:
Should probably start at zero.  Null doesn't seem to match any of the cases in the switch statement in examine.

Reply:
Null is equivalent to 0, iirc?

Reply:
Not quite. Null IS equivalent to it in logic based checks, like if(var), but you are checking number range.

Reply:
oh. Derp. I misunderstood it, then. I'll fix that.